src/components/member-card/card.module.scss

```diff
@@ -1,3 +1,21 @@
+.descriptionDivStyle {
```

Comment:
try to follow BEM for class names
https://css-tricks.com/bem-101/

Reply:
done